lib/amqp_connection.ex

```diff
@@ -15,5 +14,5 @@ defmodule Mississippi.Consumer.AMQPDataConsumer.AMQPConnection do
   Initializes the AMQP connection.
   Returns `{:ok, channel}` if it was initialized successfully, or `{:error, reason}` if it was not.
   """
-  @callback init(state :: State.t()) :: {:ok, Channel.t()} | {:error, reason :: term()}
+  @callback init(state :: any()) :: {:ok, Channel.t()} | {:error, reason :: term()}
```

Comment:
Consistency: let's use either `any()` or `term()` everywhere

Reply:
i've used `term()` in both places